data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/PostBlock.java

```diff
@@ -34,12 +35,13 @@
 import io.javalin.plugin.openapi.annotations.OpenApiContent;
 import io.javalin.plugin.openapi.annotations.OpenApiRequestBody;
 import io.javalin.plugin.openapi.annotations.OpenApiResponse;
-import org.apache.tuweni.bytes.Bytes32;
+import joptsimple.internal.Strings;
```

Comment:
er, this looks like an odd import.  I think I'd just use `""` for an empty string, seems pretty easily understandable.

Reply:
wow that is weird.. i thought strings.Empty was pretty standard, i'll fix. ta